website/docs/r/uplink_host_switch_profile.html.markdown

```diff
@@ -0,0 +1,130 @@
+  * `active` - (Required) List of Uplinks used in active list
+    * `uplink_name` - (Required) Name of this uplink
+    * `uplink_type` - (Required) Type of the uplink. Possible values are: PNIC, LAG.
+  * `policy` - (Required) Teaming policy. Possible values are: FAILOVER_ORDER, LOADBALANCE_SRCID, LOADBALANCE_SRC_MAC.
+  * `standby` - (Optional) List of Uplinks used in standby list
+    * `uplink_name` - (Required) Name of this uplink
+    * `uplink_type` - (Required) Type of the uplink. Possible values are: PNIC, LAG.
+* `named_teaming` - (Optional) List of named uplink teaming policies that can be used by logical switches
+  * `active` - (Required) List of Uplinks used in active list
+    * `uplink_name` - (Required) Name of this uplink
+    * `uplink_type` - (Required) Type of the uplink. Possible values are: PNIC, LAG.
+  * `policy` - (Required) Teaming policy. Possible values are: FAILOVER_ORDER, LOADBALANCE_SRCID, LOADBALANCE_SRC_MAC.
+  * `standby` - (Optional) List of Uplinks used in standby list
+    * `uplink_name` - (Required) Name of this uplink
+    * `uplink_type` - (Required) Type of the uplink. Possible values are: PNIC, LAG.
+  * `name` - (Optional) An uplink teaming policy of a given name defined in UplinkHostSwitchProfile. The names of all NamedTeamingPolicies in an UplinkHostSwitchProfile must be different, but a name can be shared by different UplinkHostSwitchProfiles. Different TransportNodes can use different NamedTeamingPolicies having the same name in different UplinkHostSwitchProfiles to realize an uplink teaming policy on a logical switch. An uplink teaming policy on a logical switch can be any policy defined by a user; it does not have to be a single type of FAILOVER or LOADBALANCE. It can be a combination of types, for instance, a user can define a policy with name \"MyHybridTeamingPolicy\" as \"FAILOVER on all ESX TransportNodes and LOADBALANCE on all KVM TransportNodes\".
+
+## Attributes Reference
+
+In addition to arguments listed above, the following attributes are exported:
+
+* `id` - ID of the resource.
+* `revision` - Indicates current revision number of the object as seen by NSX-T API server. This attribute can be useful for debugging.
+* `path` - The NSX path of the policy resource.
+
+## Importing
+
+An existing object can be [imported][docs-import] into this resource, via the following command:
+
+[docs-import]: https://www.terraform.io/cli/import
+
+```
+terraform import nsxt_uplink_host_switch_profile.uplink_host_switch_profile UUID
+```
+
+The above command imports UplinkHostSwitchProfile named `uplink_host_switch_profile` with the NSX ID `UUID`.
+
+
+```
+terraform import nsxt_uplink_host_switch_profile.uplink_host_switch_profile POLICY_PATH
+```
+The above command imports the uplink host switch profile named `uplink_host_switch_profile` with policy path `POLICY_PATH`.
+Note: for multitenancy projects only the later form is usable.
```

Comment:
Since multitenancy is not applicable here, lets remove this line

Reply:
Done.